src/plugins/workspace/server/plugin.ts

```diff
@@ -82,7 +93,12 @@ export class WorkspacePlugin implements Plugin<{}, {}> {
       new SavedObjectsClient(repositoryFactory.createInternalRepository())
     );
 
-    core.capabilities.registerProvider(() => ({ workspaces: { enabled: true } }));
+    core.capabilities.registerProvider(() => ({
+      workspaces: {
+        enabled: true,
+        permissionEnabled: isPermissionControlEnabled,
```

Comment:
Do we need to register this flag to the capabilities services?

Reply:
I believe @ruanyl add this here already. https://github.com/ruanyl/OpenSearch-Dashboards/blob/workspace/src/core/types/capabilities.ts#L51